src/site/content/en/blog/iframe-lazy-loading/index.md

```diff
@@ -221,22 +215,6 @@ while offering data-savings for those who are not scrolling all the way down a
 page. We are hopeful this is the first of many embeds to explore standardized iframe
 lazy-loading in production.
 
-### Wait, can't browsers just automatically lazy-load offscreen iframes?
-
-They certainly can. In Chrome 77, Chrome added support for automatically
-lazy-loading offscreen images and iframes when a user has opted into
-[Lite Mode](https://blog.chromium.org/2019/04/data-saver-is-now-lite-mode.html)
-(Data Saver mode) in Chrome for Android.
-
-Lite Mode is commonly used in regions of the world where network connection
-quality and data plans are not the greatest. Every byte matters and so
-lazy-loading iframes has the potential to make a meaningful difference for
-these users.
-
-Origins can detect what percentage of their traffic is coming from Lite Mode
-users by checking the `navigator.connection.saveData` property,
-which is part of the [`NetworkInformation` API](https://developer.mozilla.org/docs/Web/API/NetworkInformation).
-
```

Comment:
I am wondering if we should preserve this section as more of a historic FAQ item. My thinking is that it's possible a reader will have heard in previous years that there are situations where Chrome may have automatically lazy-loaded and a perspective on this ("we did, but alas no more") may be helpful. 

That said, I'm 100% happy to go for group preference here and if others don't feel strongly about keeping it I am happy for it to be dropped.